Gems/Multiplayer/Code/Include/Multiplayer/NetworkTime/RewindableObject.h

```diff
@@ -64,6 +64,10 @@ namespace Multiplayer
         //! @return value in const base type form
         const BASE_TYPE& GetPrevious() const;
 
+        //! Const base type retriever for value at last known authoritative update or the nearest to it.
+        //! @return value in const base type form
+        const BASE_TYPE& GetAuthority() const;
```

Comment:
This guy is a bit concerning. I think it might be worth having another look at. m_authTime is only updated on a serialize call, which is a bit undefined to be honest (could be a correction, could be a property update, etc..) and is zero at all other times. Also I don't know what the behaviour should be on a client vs a server.

Coupling to a role like authority at this level seems a bit wrong. You might want to back this up to 'GetLastSerializedValue()' or something a bit more generalized.

Reply:
I'll take a deeper look through this.